frontend/services/viewer/viewer.tsx

```diff
@@ -477,23 +466,101 @@ export class ViewerService {
 	}
 
 	public measurementAlertEvent(measurement) {
+		if (this.addedMeasurements.has(measurement.uuid)) {
```

Comment:
do we still need these stuff if unity is no longer giving you an event for every measurement added?

Not a big fan of having this state on the service.

Reply:
N we dont, I forgot  to take it off after Sebastian's change 😅